setup.py

```diff
@@ -57,6 +57,8 @@ def readme():
     # Sentry: Log Aggregation
     'raven >= 5.1.1',
     'blinker >= 1.3',
+    # Deployment
+    "invoke == 0.9.0",
```

Comment:
How about add `invoke` to a separated new list e.g. `deploy_requires`?  If you register these extra requirements to `extras_require` e.g. `extras_require={'tests': tests_require, 'docs': docs_require, 'deploy': deploy_requires}` you would be able to install them by `pip install -e .[deploy]` command.